stylesheet.css

```
@@ -0,0 +1,127 @@
+@charset "UTF-8";
+
+* {
+    box-sizing: border-box;
+}
+
+/* html {
+    font-size: 100%;
+} */
+
+body {
+    color: #2b2a27;
+    font-family: "Helvetica Neue", "Arial", "Hiragino Sans", "Meiryo", sans-serif;
+}
+
+img {
+    max-width: 100%;
+}
+
+ul {
+    list-style: none;
+}
+
+a {
+    color: #2b2a27;
+}
+
+header {
+    margin: 0 auto;
+    max-width: 960px;
+    /* max-width: 100%; */
+    padding: 121px 4% 120px 4%;
+}
+
+.header-txt {
+
+    text-align: center;
+    justify-content: space-between;
+    display: flex;
+}
+
+.header-txt img {
+    vertical-align: left;
+    width: 120px;
+    height: 27px;
+    /* text-align: left; */
+}
```

上下中央揃えにして、imgの高さ増やしてみたけれどどう？なんとなくデザインと合うと思うけれど
```suggestion
.header-txt {

    text-align: center;
    justify-content: space-between;
    align-items: center;  // 追加
    display: flex;
}

.header-txt img {
    vertical-align: left;
    width: 120px;
    height: 36px;　 // 修正
    /* text-align: left; */
}
```